src/main/java/com/conveyal/analysis/spatial/Polygons.java

```diff
@@ -0,0 +1,9 @@
+package com.conveyal.analysis.spatial;
+
+public class Polygons {
```

Comment:
What is the intended use of these classes (Polygons, Points, Lines)? Or are they just left over from a draft / prototype?

Reply:
Leftovers, possible container for static utility methods.